### PR TITLE
recreates mhv_opt_in_flags table

### PR DESCRIPTION
**Pull request by bramleyjl**
## Description of change

This change updates the [original PR](https://github.com/department-of-veterans-affairs/vets-api/pull/10711) to add the `mhv_opt_in_flags_table`

## Original issue(s)
department-of-veterans-affairs/va.gov-team#46578

### CODE REVIEW

File: db/migrate/20220916043854_delete_and_recreate_mhv_opt_in_flags.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class DeleteAndRecreateMHVOptInFlags < ActiveRecord::Migration[6.1]
+  def up
+    drop_mhv_opt_in_flags_table
+    create_mhv_opt_in_flags_table
+  end
+
+  def down
+    drop_mhv_opt_in_flags_table
+    create_mhv_opt_in_flags_table
```
Comment (bosawt) [inline db/migrate/20220916043854_delete_and_recreate_mhv_opt_in_flags.rb:9]: For `down`, you should use the definition from the previous migration, just to keep it consistent:

>     create_table :mhv_opt_in_flags do |t|
>      t.string :user_account_uuid, null: false
>       t.string :opt_in_flag, null: false
>
>       t.index [ "user_account_uuid" ],  unique: false
>       t.index [ "opt_in_flag" ],  unique: false
>       t.timestamps
>     end

File: db/migrate/20220916043854_delete_and_recreate_mhv_opt_in_flags.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+class DeleteAndRecreateMHVOptInFlags < ActiveRecord::Migration[6.1]
+  def up
+    drop_mhv_opt_in_flags_table
+    create_mhv_opt_in_flags_table
+  end
+
+  def down
```
Comment (bosawt) [inline db/migrate/20220916043854_delete_and_recreate_mhv_opt_in_flags.rb:7]:
```suggestion
  def down
    drop_mhv_opt_in_flags_table
```

Comment (bosawt) [reply db/migrate/20220916043854_delete_and_recreate_mhv_opt_in_flags.rb:7]: For the 'down' you'll need to drop table too or it'll freak out when trying to recreate the table from the previous merge

**Review by bosawt — APPROVED** [review]
Nice! Confirmed that `rails db:migrate` and then `rails db:rollback` both work and have expected results

**Review by LindseySaari — APPROVED** [review]

**Pull request merged**